content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/lifo.rst

```diff
@@ -0,0 +1,64 @@
+============
+LIFO removal
+============
+
+The *last in, first out* (LIFO) removal strategy picks the **newest** products on-hand based on the
+date they entered a warehouse's stock.
+
+Every time an order for products with the :abbr:`LIFO (Last In, First Out)` method is placed, a
+transfer is created for the lot/serial number that has most recently entered the stock (the **last**
+lot/serial number that entered the warehouse's inventory).
+
+.. warning::
+   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy in banned, since it can
+   potentially result in old, expired, or obsolete products being delivered to customers.
```

Comment:
```suggestion
   In many countries, the :abbr:`LIFO (Last In, First Out)` removal strategy is banned, since it can
   potentially result in old, expired, or obsolete products being delivered to customers.
```